test/unittest/tests/block/test_extent_list_search.cpp

```diff
@@ -0,0 +1,296 @@
+        };
+
+        /* Setup. */
+        WT_EXTLIST extlist = {};
+
+        /* Insert extents. */
+        for (const off_size &to_insert : insert_list) {
+
+            INFO("Insert: {off " << std::showbase << to_insert.off << ", size " << to_insert.size
+                                 << ", end " << to_insert.end() << "}");
+            REQUIRE(__ut_block_off_insert(session, &extlist, to_insert.off, to_insert.size) == 0);
+        }
+
+        extlist_print_off(extlist);
+
+        /* Test. */
+        WT_EXT dummy;
+        uint32_t idx = 0;
+        for (const search_before_after &expected : expected_before_after) {
+            WT_EXT *before = &dummy;
+            WT_EXT *after = &dummy;
+            /* Call. */
+            __ut_block_off_srch_pair(&extlist, expected.off, &before, &after);
+
+            std::ostringstream line_stream;
+            line_stream << "Verify: " << idx << ". off " << expected.off;
+            if (expected.before != nullptr)
+                line_stream << "; Expected: before: {off " << expected.before->off << ", size "
+                            << expected.before->size << ", end " << expected.before->end() << "}";
+            else
+                line_stream << "; Expected: before == nullptr";
+
+            if (expected.after != nullptr)
+                line_stream << ", after: {off " << expected.after->off << ", size "
+                            << expected.after->size << ", end " << expected.after->end() << "}";
+            else
+                line_stream << ", after == nullptr";
+
+            if (before != nullptr)
+                line_stream << "; Actual: before: {off " << before->off << ", size " << before->size
+                            << ", end " << (before->off + before->size - 1) << "}";
+            else
+                line_stream << "; Actual: before == nullptr";
+
+            if (after != nullptr)
+                line_stream << ", after: {off " << after->off << ", size " << after->size
+                            << ", end " << (after->off + after->size - 1) << "}";
+            else
+                line_stream << ", after == nullptr";
+
+            std::string line = line_stream.str();
+            INFO(line);
+
+            /* Verify. */
+            if (expected.before != nullptr) {
+                REQUIRE(before != nullptr);
+                REQUIRE(before->off == expected.before->off);
+                REQUIRE(before->size == expected.before->size);
+            } else
+                REQUIRE(before == nullptr);
+
+            if (expected.after != nullptr) {
+                REQUIRE(after != nullptr);
+                REQUIRE(after->off == expected.after->off);
+                REQUIRE(after->size == expected.after->size);
+            } else
+                REQUIRE(after == nullptr);
+            ++idx;
+        }
+
+        /* Cleanup. */
+        extlist_free(session, extlist);
+    }
+}
+
+#ifdef HAVE_DIAGNOSTIC
+/*!
```

Comment:
I think `/*!` is a documentation style comment format, typically WT does not use that style outside of WiredTiger.in. Do you have a reason for opting for it?

Reply:
I copied it from WiredTiger.in because I wanted a documentation style comment. How do we do documentation style comments in the rest of WiredTiger?

Reply:
We just use standard comments.